fdbserver/DataDistribution.actor.cpp

```diff
@@ -408,6 +628,23 @@ struct DataDistributor : NonCopyable, ReferenceCounted<DataDistributor> {
 			wait(self->takeMoveKeysLock());
 			TraceEvent("DDInitTookMoveKeysLock", self->ddId).log();
 
+			// AuditStorage does not rely on DatabaseConfiguration
+			// AuditStorage read neccessary info purely from system key space
+			wait(self->initAuditStorage(self));
```

Comment:
We don't want to `wait` here, can it be asynchronous?

Reply:
Yes. Changed